src/utils/require-template.js

```diff
@@ -6,24 +6,28 @@ const path = require('path');
 const templateNotFound = 'Error requiring oc-template: "{0}" not found';
 const templateNotValid = 'Error requiring oc-template: "{0}" is not a valid oc-template';
 
-function isValidTemplate(template){
+function isValidTemplate(template, options){
   if (typeof template !== 'object') {
     return false;
   }
 
+  if (options.compiler === true) {
```

Comment:
Is not a function but a boolean that can be passed as an option at call time. need to add it [here](https://github.com/opentable/oc/blob/2485f6a623d884af311a8e4e60f0383316dad74d/src/cli/domain/package-template.js#L25)

Does make sense?